src/shogun/lib/SGMatrix.cpp

```diff
@@ -407,6 +407,12 @@ T* SGMatrix<T>::clone_matrix(const T* matrix, int32_t nrows, int32_t ncols)
 	return result;
 }
 
+template <class T>
+SGMatrix<T> SGMatrix<T>::empty_like(const SGMatrix<T>& other)
```

Comment:
what about just `empty`?

Reply:
sure, this was just to keep it in line with numpy

Reply:
ah I see. Think that is good

Reply:
shall I merge this like it is now?